### ISSUE TITLE
Breaking changes ? In general or possibly windows binary build.

### ISSUE THREAD

**Issue by juliandewit** [dmlc/mxnet#1266]
Hello mxnet suddenly stopped working between builds

Converges for binary build :
20151209 and 20151228

No learning happens for binary build:
20160106 and 20160113

I took the MNIST example as a reference but other example also showed no learning.
With mnist I tried 'lenet' and 'mlp'. Both did not work. Tried examples with softmax and regression.
All had the same phenomenom.

I checked the dll libraries that were loaded and I did not see anything suspicous.
See the attached image. I cleaned the python dir and reinstalled.
All to no avail.

Does anyone know about a possible breaking change between the 20151228 and the 20160106 release ?

Update 1:
The MNIST example works for the CPU. So it has to do with the GPU code.

![loaded_libraries](https://cloud.githubusercontent.com/assets/8937693/12443042/34bcba30-bf55-11e5-97ec-dcb44b79522a.png)

**Comment by juliandewit** [comment]
Excuse me.. There might be something more to it.
The MNIST example also does not converge for lenet and mlp..
Closing for now

**Comment by hjk41** [comment]
I just hit the same problem with MxNet.cpp. I thought it was the cpp wrapper that is causing the problem. Let me try to figure this out.

**Comment by juliandewit** [comment]
Thanks for the quick reply:

I ruled out a few things:
- It works for CPU
- I isolated/left out many layer/symbol types :
  Convolution (problem is also when I don't use them)
  Fully Connected  (problem is also when I don't use them)
  BatchNorm (problem is also when I don't use them)
  Softmax (problem is also when I don't use them)
- The newest release did work on a completely clean other windows system.
  However.. looking at the loaded libraries it seems exactly the same

**Comment by hjk41** [comment]
@juliandewit We are now trying to diagnose the problem in #1228 . Let's move the discussion there. I am closing this issue for now.
